SU-SWS/linky_clicky
# SOE-1792: Adding engineering-redesign

# READY FOR REVIEW

# Summary
- Branch for developing the engineering redesign (phase 2)

# Needed By (Date)
- For SoE Phase 2 launch

# Urgency

# Steps to Test

Pull this branch ($ git checkout soe-redesign)
Navigate to linky_clicky/sites/engineering-redesign
Run 'live' tests on an Engineering redesigned site, e.g. $ ../../bin/behat  -p \<your-site-alias\> -s live features
Verify no tests fail unexpectedly.

# Affected Projects or Products
- SoE Redesign

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1792
- https://stanfordits.atlassian.net/browse/SOE-2153
- Review ticket:  https://stanfordits.atlassian.net/browse/SOE-2154
- Who should be notified: @josephgknox, @boznik, David Tom

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
